experimental/traffic-portal/src/environments/environment.prod.ts

```diff
@@ -21,5 +21,6 @@ import type { Environment } from "./environment.type";
 export const environment: Environment = {
 	apiVersion: "4.0",
 	customModule: false,
-	production: true
+	production: true,
+	tpv1Url: "https://tp.staging.cdnlab.comcast.net/#!",
```

Comment:
You shouldn't link this to any actual environments, especially any comcast internal ones. I'd have the default point to one of the CiaB instances. Either `localhost:444` or `localhost:443`